extras/installer.sh

```diff
@@ -186,6 +186,8 @@ configure_plexupdate() {
 		[ -z "$DISTRO" ] && check_distro
 		if [ "$DISTRO" == "redhat" ]; then
 			AUTOSTART=yes
+		elif [ "$DISTRO" == "synology" ]; then
```

Comment:
I have a pathological aversion to duplicating code, so I'd write this as `if [ "$DISTRO" == "redhat" -o "$DISTRO" == "synology" ]; then`.